static/src/components/TaskCard.vue

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+import type { Task } from "@/api/models/Task";
+export default defineComponent({
+  props: {
+    task: { type: Object as () => Task, required: true },
+  },
+
+  emits: ["annotate"],
+});
+</script>
+
+<template>
+  <i-card>
+    <template #header>
+      {{ task.id }}
+    </template>
+    {{ task.description }}
+    <template #footer>
+      <i-button color="primary" @click="$emit('annotate', task)">
```

Comment:
Nice-to-have: I would suggest renaming this function to something like `open_task` or `start_session/task`, as this button doesn't have anything to do with an actual annotation yet. But I'm also open to your thoughts or just keeping it like this :)

Reply:
`open_task` sound nice for me :) 👍